db/cache/charts.go

```diff
@@ -931,9 +932,11 @@ func coinSupplyChart(charts *ChartData, zoom ZoomLevel) ([]byte, error) {
 func durationBTWChart(charts *ChartData, zoom ZoomLevel) ([]byte, error) {
 	switch zoom {
 	case BlockZoom:
-		return charts.encode(charts.Blocks.Time, btw(charts.Blocks.Time))
+		t, d := btw(charts.Blocks.Time)
+		return charts.encode(t, d)
```

Comment:
No problem, but note that go will bind the return values of `btw` to the parameters of `charts.encode` if you do:

```go
charts.encode(btw(charts.Blocks.Time))
```

(not just because `charts.encode` is variadic, but because the spec allows this kind of thing.  https://play.golang.org/p/8mPe9bDAT-x

Reply:
that sweet sugar! I'm gonna update it.